contents/docs/billing/limits-alerts.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Billing limits and alerts
+---
+
+To help you avoid surprise bills, PostHog lets you set billing limits for each of our products. To do so:
+
+1. Go to your organization's [billing settings](https://app.posthog.com/organization/billing)
+2. Click on the three dots in the top right of a product, then "Set billing limit."
+3. Set your dollar limit in the box at the bottom of the product and press "Save."
+
+You’ll need to do this for each of the products. You can also remove limits by following the same process and clicking “Remove limit” instead of "Save."
+
+![two types of cohorts: static and dynamic](../../images/docs/billing/alerts.png)
+
+## Billing alerts
+
+When you set billing limits, the owner of the organization will automatically get alert emails when product usage nears the billing limits. These emails are sent at **80%** and **100%** of the limit.
```

Comment:
We also send these emails when people are on the free plan when they reach 80% and 100% of the free allotment.